File: codes_Lecture_01/p_1_5_1_3.py
# 错误的缩进示例
# NOTE 运行本脚本文件会报错（IndentationError）

year = 2024

if year % 400 == 0 or (year % 4 == 0 and year % 100 != 0):
    print(year, "年是闰年")
else:
print(year, "年不是闰年")

File: codes_Lecture_01/p_1_5_1_3.cpp
#include <iostream>

int main()
{
    int year = 2024;

    if (year % 400 == 0 or (year % 4 == 0 and year % 100 != 0))
    {
        std::cout << year << " is a gap year." << std::endl;
    }
    else
    {
std::cout << year << " is NOT a gap year." << std::endl;        
    } // C++语言中 通过花括号{}确定层次结构 对缩进无强制要求 

    return 0;
}